frontend/packages/shared/src/api/queries.ts

```diff
@@ -76,7 +78,7 @@ import type { WidgetSettingsResponse } from 'app-shared/types/widgetTypes';
 import { buildQueryParams } from 'app-shared/utils/urlUtils';
 import { orgListUrl } from '../cdn-paths';
 import type { JsonSchema } from 'app-shared/types/JsonSchema';
-import type { PolicyAction, PolicySubject } from '@altinn/policy-editor';
+import type { PolicyAction, PolicySubject, AccessPackageResource, PolicyAccessPackageAreaGroup } from '@altinn/policy-editor';
```

Comment:
We should not add more technical debt when we introduce new code 😄 
I understand that the two first types imported are not related to changes in this PR, but we should not introduce more debt by importing two new types from a packages into shared. 
The two other ones can be fixed in a separate PR as they probably will change more than one place due to imports, but the two new ones should not be imported from the package.